### PR TITLE
Pending report bug fixes

### PR DESCRIPTION
**Pull request by matthopson**

### CODE REVIEW

File: shared/src/business/useCases/pendingItems/generatePrintablePendingReportInteractor.js [hunk 1]
```diff
@@ -34,11 +34,14 @@ exports.generatePrintablePendingReportInteractor = async ({
 
   const formattedPendingItems = pendingItems.map(pendingItem => ({
     ...pendingItem,
-    associatedJudgeFormatted: pendingItem.associatedJudge.replace(
+    associatedJudgeFormatted: (pendingItem.associatedJudge || '').replace(
```
Comment (kkoskelin) [inline shared/src/business/useCases/pendingItems/generatePrintablePendingReportInteractor.js]: 🍯 🦡

**Review by kkoskelin — APPROVED** [review]

File: shared/src/business/useCases/pendingItems/generatePrintablePendingReportInteractor.js (hunk 1 above)
Comment (matthopson) [reply shared/src/business/useCases/pendingItems/generatePrintablePendingReportInteractor.js]: I dunno what that means.

**Comment by matthopson** [comment]
Probably help if I push this ☝️

File: shared/src/business/useCases/pendingItems/generatePrintablePendingReportInteractor.js (hunk 1 above)
Comment (kkoskelin) [reply shared/src/business/useCases/pendingItems/generatePrintablePendingReportInteractor.js]: Honey Badger. I think you were fixing some things that complained on HB. :)

**Review by kkoskelin — APPROVED** [review]

File: shared/src/business/useCases/pendingItems/generatePrintablePendingReportInteractor.js (hunk 1 above)
Comment (matthopson) [reply shared/src/business/useCases/pendingItems/generatePrintablePendingReportInteractor.js]: Ah, that makes sense. GH doesn't have alt tags for emojis, so hovering was no help. Go figure....

**Pull request merged**